src/WebJobs.Script/runtimeassemblies-v3.json

```diff
@@ -843,10 +839,6 @@
       "name": "System.Diagnostics.Debug",
       "resolutionPolicy": "minorMatchOrLower"
     },
-    {
-      "name": "System.Diagnostics.DiagnosticSource",
-      "resolutionPolicy": "minorMatchOrLower"
-    },
     {
```

Comment:
Do we also need to remove these? https://github.com/Azure/azure-functions-host/pull/4941/files

`"name": "System.Runtime.CompilerServices.Unsafe",`

`"name": "System.Text.Encodings.Web",`

Reply:
Wondering why those are missing.... they should have been merged into the 2.0 compat file